sources/packages/backend/apps/api/src/services/designation-agreement/designation-agreement.service.ts

```diff
@@ -67,7 +77,29 @@ export class DesignationAgreementService extends RecordDataModelService<Designat
         return newLocation;
       },
     );
-    return this.repo.save(newDesignation);
+    const institution = await this.institutionRepo.findOne({
+      select: {
+        operatingName: true,
+        legalOperatingName: true,
+        primaryEmail: true,
+      },
+      where: { id: institutionId },
+    });
+    const ministryNotification: InstitutionRequestsDesignationNotificationForMinistry =
+      {
+        institutionName: institution.legalOperatingName,
```

Comment:
Is `institutionName` part of the template?

Reply:
Yes